kubeflow/apps/pipelines/pull-upstream.sh

```diff
@@ -17,8 +17,8 @@
 set -ex
 
 # TODO: Use kubeflow/pipelines once https://github.com/kubeflow/pipelines/pull/6595 is resolved.
-export KUBEFLOW_PIPELINES_VERSION=2.0.0-alpha.3
-export KUBEFLOW_PIPELINES_REPO=https://github.com/kubeflow/pipelines.git
+export KUBEFLOW_PIPELINES_VERSION=$2
+export KUBEFLOW_PIPELINES_REPO=$1
```

Comment:
Would you like to add some instruction about the `repo` and `version` parameters when calling this script?

Reply:
Good point. Added. Thanks!